flink-staging/flink-streaming/flink-streaming-core/src/main/java/org/apache/flink/streaming/api/datastream/IterativeDataStream.java

```diff
@@ -81,56 +81,56 @@ protected IterativeDataStream(DataStream<T> dataStream, long maxWaitTime) {
 	/**
 	 * Changes the feedback type of the iteration and allows the user to apply
 	 * co-transformations on the input and feedback stream, as in a
-	 * {@link ConnectedDataStream}.
+	 * {@link ConnectedStreams}.
 	 *
 	 * <p>
 	 * For type safety the user needs to define the feedback type
 	 * 
 	 * @param feedbackTypeString
 	 *            String describing the type information of the feedback stream.
-	 * @return A {@link ConnectedIterativeDataStream}.
+	 * @return A {@link ConnectedIterativeDataStreams}.
 	 */
-	public <F> ConnectedIterativeDataStream<T, F> withFeedbackType(String feedbackTypeString) {
+	public <F> ConnectedIterativeDataStreams<T, F> withFeedbackType(String feedbackTypeString) {
 		return withFeedbackType(TypeInfoParser.<F> parse(feedbackTypeString));
 	}
 
 	/**
 	 * Changes the feedback type of the iteration and allows the user to apply
 	 * co-transformations on the input and feedback stream, as in a
-	 * {@link ConnectedDataStream}.
+	 * {@link ConnectedStreams}.
 	 *
 	 * <p>
 	 * For type safety the user needs to define the feedback type
 	 * 
 	 * @param feedbackTypeClass
 	 *            Class of the elements in the feedback stream.
-	 * @return A {@link ConnectedIterativeDataStream}.
+	 * @return A {@link ConnectedIterativeDataStreams}.
 	 */
-	public <F> ConnectedIterativeDataStream<T, F> withFeedbackType(Class<F> feedbackTypeClass) {
+	public <F> ConnectedIterativeDataStreams<T, F> withFeedbackType(Class<F> feedbackTypeClass) {
 		return withFeedbackType(TypeExtractor.getForClass(feedbackTypeClass));
```

Comment:
Why ConnectedIterativeDataStreams and not ConnectedIterativeStreams (following the naming of the other classes)?